pylint/lint/pylinter.py

```diff
@@ -175,8 +175,8 @@ def make_options():
                     "metavar": "<pattern>[,<pattern>...]",
                     "dest": "black_list_re",
```

Comment:
Shouldn't this be changed too ?

Reply:
Renaming the `dest` names (`black_list`, `black_list_re`, `extension_pkg_whitelist` etc.) could be breaking for plugins and other programmatic access since these are the member names in the `config` object. doc/how_tos/plugins.rst even sets `black_list` in the example.

It might be possible to clean this up in a way that's still backwards compatible -- I'll need to study optparse further since I'm not familiar with it. If `optparse.Values` doesn't provide a built-in way to alias the names, we could probably extend `optparse.Values` and add some `@property` annotations or `__getattr__/__setattr__` to provide compatibility. Or as you've noted previously, maybe it's time for bigger rework of the config system. But because it won't make a difference to most users, this seems like something to discuss and address in a later issue/PR.

Reply:
Sorry, lack of sleep, you're right, we should not touch it (not before 3.0 at least).